_includes/package_body.html

```diff
@@ -83,6 +83,49 @@ <h4>Third-Party Package</h4>
                     {{package.data.pkg_type | upcase}}</span>
                 </td>
               </tr>
+              <tr>
+                <td class="text-right"><b>CI status</b></td>
```

Comment:
This should be pushed down to the repository summary as all of our CI tests actually run on the repository not on the package level.

Reply:
Oh, I see. In this case it was wrong to search for the package name and not the repo name.

I changed that and moved it down to the other box, but now I'm not sure how to deal with packages that don't have a `devel_jobs/.../results.yaml` on the doc server - which are quite a lot.
The wiki code just seems to give them a CI badge anyways.
I thought this was a bug / missing feature and made it look for the `api/.../manifest.yaml`.
What is the expected behavior for packages that are on the build farm and have build success or failure recorded in their `manifest.yaml`, but no `results.yaml` for their repo?

Right now, the flow is:
(1) check for `manifest.yaml` (package-base)
If none is found, show "No CI" bagde.
(2) check for `results.yaml` (repository based)
If none is found, show simple CI badge with no additional information.
(3) parse `results.yaml` for additional information and display extended badge with test results if available.